packages/kbn-plugin-generator/sao_template/template/translations/zh-CN.json

```diff
@@ -1,84 +0,0 @@
-{
```

Comment:
@lizozom are we considering to re-add a translation example to the plugin generator?

Reply:
I don't know how to :)
How is this going to work in NP?

Reply:
@joshdover what is your opinion here?

Reply:
Seems like this wouldn't work without a considerable hack.

Reply:
Maybe we can add this in a follow up PR if everything else is a'ight?